sublime/User/Package Control.sublime-settings

```diff
@@ -5,39 +5,47 @@
 	],
 	"installed_packages":
 	[
+		"A File Icon",
 		"All Autocomplete",
 		"Better CoffeeScript",
-		"Color Highlighter",
-		"DashDoc",
-		"Delete Current File",
 		"Diffy",
-		"EncodingHelper",
-		"File Rename",
+		"Ecmascript Syntax",
+		"Edge Theme",
+		"Elixir",
+		"ElixirSyntax",
+		"FileManager",
 		"Git",
-		"GitGutter",
 		"Jade",
-		"Markdown Preview",
-		"Materialize",
+		"Javascript Beautify",
+		"JavaScriptNext - ES6 Syntax",
+		"Laravel Blade Highlighter",
+		"MarkdownPreview",
 		"Open Folder",
 		"Package Control",
 		"Path Tools",
 		"PHP Haml",
+		"Pug",
 		"Rails Latest Migration",
 		"Ruby Hash Converter",
 		"Ruby Slim",
 		"RubyTest",
 		"Sass",
-		"SimpleCov",
 		"Stylus",
 		"SublimeLinter",
 		"SublimeLinter-coffee",
+		"SublimeLinter-coffeelint",
 		"SublimeLinter-contrib-scss-lint",
+		"SublimeLinter-contrib-slim-lint",
+		"SublimeLinter-eslint",
 		"SublimeLinter-haml",
 		"SublimeLinter-json",
 		"SublimeLinter-php",
 		"SublimeLinter-rubocop",
+		"SublimeLinter-stylelint",
 		"SublimeLinter-xmllint",
-		"Terminal",
+		"Svelte",
+		"Svelte Snippets",
```

Comment:
Would it be affordable to exclude this from the PR? Or it will be easier to iterate with a new commit?

Asking about it 'cos we don't - yet? - use svelte in weLaika ;)